src/BinaryParsers/PEBinary/ProgramDatabase/ObjectModuleDetails.cs

```diff
@@ -224,28 +224,35 @@ public string GetDialect(out string versionNumber)
         }
     }
 
+    /// <summary>
+    /// Information about the CV_CFL_LANG enumeration, 
+    /// which specifies the code language of the application or linked module in the debug interface access SDK.
+    /// https://docs.microsoft.com/en-us/visualstudio/debugger/debug-interface-access/cv-cfl-lang
+    /// </summary>
     public enum Language : uint
     {
         C = 0x00,
         Cxx = 0x01,
         FORTRAN = 0x02,
-        MASM = 0x03,
+        MASM = 0x03, // Microsoft Macro Assembler
         Pascal = 0x04,
         Basic = 0x05,
         COBOL = 0x06,
-        LINK = 0x07,
-        CVTRES = 0x08,
-        CVTPGD = 0x09,
+        LINK = 0x07, // linker-generated module
+        CVTRES = 0x08, // resource module converted with CVTRES tool
+        CVTPGD = 0x09, // POGO optimized module generated with CVTPGD tool
         CSharp = 0x0A,
         VisualBasic = 0x0B,
-        ILASM = 0x0C,
+        ILASM = 0x0C, // intermediate language assembly (that is, Common Language Runtime (CLR) assembly)
         Java = 0x0D,
         JScript = 0x0E,
-        MSIL = 0x0F,
+        MSIL = 0x0F, // Microsoft Intermediate Language (MSIL), possibly a result of using the /LTCG (Link-time Code Generation) switch
         HLSL = 0x10, // High Level Shader Language
         ObjectiveC = 0x11,
         ObjectiveCxx = 0x12,
         Swift = 0x13,
+        ALIASOBJ = 0x14, // module generated by the aliasobj tool
+        Rust = 0x15,
```

Comment:
>Rust [](http://example.com/codeflow?start=8&length=4)

Note that for this language, you didn't say 'Module generated by Rust.' So, in general, please be consistent in your approach. I don't think this comment would provide much value, by the way. If your comment doesn't add useful additional information, we are just cluttering the code.

Reply:
OK, so I think what you've done is to partly port the documentation for enum values over. My feedback to you is: don't do that in an inconsistent way. 

You have two choices:

1) Author useful comments in places where you think a comment provides additional information over the language name. Is anyone confused by what 'Pascal' means here, for example?
2) OR: port all the comments from the documentation, so that every member is consistently documented precisely as it is online.

i.e., stay out of the middle, which is to inconsistently apply exact language in some cases, partial documentation in others, and to ignore others entirely.

V_CFL_C Application language is C.

CV_CFL_CXX Application language is C++.

CV_CFL_FORTRAN Application language is FORTRAN.

CV_CFL_MASM Application language is Microsoft Macro Assembler.

CV_CFL_PASCAL Application language is Pascal.

CV_CFL_BASIC Application language is BASIC.

CV_CFL_COBOL Application language is COBOL.

CV_CFL_LINK Application is a linker-generated module.

CV_CFL_CVTRES Application is a resource module converted with CVTRES tool.

CV_CFL_CVTPGD Application is a POGO optimized module generated with CVTPGD tool.

CV_CFL_CSHARP Application language is C#.

CV_CFL_VB Application language is Visual Basic.

CV_CFL_ILASM Application language is intermediate language assembly (that is, Common Language Runtime (CLR) assembly).

CV_CFL_JAVA Application language is Java.

CV_CFL_JSCRIPT Application language is Jscript.

CV_CFL_MSIL Application language is an unknown Microsoft Intermediate Language (MSIL), possibly a result of using the /LTCG (Link-time Code Generation) switch.

CV_CFL_HLSL Application language is High Level Shader Language.

CV_CFL_OBJC Application language is Objective-C.

CV_CFL_OBJCXX Application language is Objective-C++.

CV_CFL_SWIFT Application language is Swift.

CV_CFL_ALIASOBJ Application is a module generated by the aliasobj tool.

CV_CFL_RUST Application language is Rust.

Reply:
stay out of the middle --- agree

and also agree some of the comments does not add value at all, so removed all to be consistent.